### PR TITLE
Submission attr

### PR DESCRIPTION
**Pull request by phlax**
ensure correct attribution and submission times in store update

### CODE REVIEW

File: pootle/apps/pootle_store/updater.py [hunk 1]
```diff
@@ -153,6 +153,12 @@ def should_update_source(self):
                 and self.unit.source != self.newunit.source)
 
     @cached_property
+    def resurrected(self):
+        return (
+            self.newunit
+            and self.unit.isobsolete() != self.newunit.isobsolete())
```
Comment (ta2-1) [inline pootle/apps/pootle_store/updater.py]: this condition will be `True` for opposite change as well. Probs we need to be explicit here or just rename the method.

File: pootle/apps/pootle_store/updater.py [hunk 2]
```diff
@@ -222,20 +228,23 @@ def set_unit(self):
...
             and self.update.store_revision is not None
             and (self.update.store_revision > self.unit.revision
                  and not self.update.resolve_conflict == POOTLE_WINS)
             and not self.newunit.isobsolete())
 
     def update_unit(self):
+        reordered = False
         suggested = False
         updated = False
         need_update = (self.should_unobsolete
```
Comment (unho) [inline pootle/apps/pootle_store/updater.py]: Unrelated, but `unobsolete` probably should be renamed to `resurrect`.

File: pootle/apps/pootle_store/updater.py [hunk 3]
```diff
@@ -222,20 +228,23 @@ def set_unit(self):
             self.set_commented()
         if self.target_updated:
             self.set_submitted()
+        if self.resurrected:
```
Comment (unho) [inline pootle/apps/pootle_store/updater.py]: Nit: fold in previous `if`.

File: pootle/apps/pootle_store/updater.py (hunk 1 above)
Comment (unho) [inline pootle/apps/pootle_store/updater.py]: This comparison might return True when unit is obsoleted.

Comment (unho) [inline pootle/apps/pootle_store/updater.py]: The condition needs to be changed. It returns `True` when unit is obsoleted.

File: pootle/apps/pootle_store/updater.py [hunk 4]
```diff
@@ -153,6 +153,13 @@ def should_update_source(self):
                 and self.unit.source != self.newunit.source)
 
     @cached_property
+    def resurrected(self):
+        return (
+            self.newunit
+            and self.newunit.isobsolete()
+            and self.unit.isobsolete() != self.newunit.isobsolete())
```
Comment (unho) [inline pootle/apps/pootle_store/updater.py]: Nit: `and not self.unit.isobsolete()`

**Comment by unho** [comment]
lgtm

File: pootle/apps/pootle_store/updater.py (hunk 4 above)
Comment (phlax) [inline pootle/apps/pootle_store/updater.py]: i got this the wrong way round - but i got what you meant - and updated...

**Pull request merged**